File: official/docs/ruby/current/batches/add-shipments.rb
require 'easypost'

client = EasyPost::Client.new(api_key: 'EASYPOST_API_KEY')

retrieved_batch = client.batch.retrieve('batch_...')

batch = client.batch.add_shipments(
  retrieved_batch.id,
  shipments: [
    { id: 'shp_...' },
    { id: 'shp_...' },
  ],
)

puts batch

File: official/docs/node/current/batches/add-shipments.js
const EasyPostClient = require('@easypost/api');

const client = new EasyPostClient('EASYPOST_API_KEY');

(async () => {
  const batch = await client.Batch.addShipments('batch_...', ['shp_...', 'shp_...']);

  console.log(batch);
})();
